packages/aws-cdk/lib/context-providers/vpcs.ts

```diff
@@ -71,7 +71,7 @@ export class VpcNetworkContextProviderPlugin implements ContextProviderPlugin {
         throw new Error(`Subnet ${subnet.SubnetArn} has invalid subnet type ${type} (must be ${SubnetType.Public}, ${SubnetType.Private} or ${SubnetType.Isolated})`);
```

Comment:
A user has in some issue reported that subnet type deduction "didn't work" (no more information than that). I guess you haven't looked at that, have you?

Reply:
I'm working off of issue #3407, and I don't see any mention of subnet type deduction not working there. Do you have the issue where that concern was raised? We can ask the customer to provide us the exact subnets that caused the deduction to fail with `ec2 describe-subnets` like we did in #3407 .

Reply:
I'm sorry, I can't find it anymore. It's one of the various threads mentioning VPCs but I can't tell which one anymore.